addons/elasticsearch-cluster/templates/cluster.yaml

```diff
@@ -39,16 +39,3 @@ spec:
               requests:
                 storage: {{ .Values.persistence.data.size }}
       {{- end }}
-
-    - name: metrics # user-defined
-      componentDefRef: metrics # ref clusterdefinition componentDefs.name
-      replicas: {{ .Values.metrics.replicaCount | default 1 }}
```

Comment:
the values of metrics should also be cleared

Reply:
done